.travis/clone_build_and_install_libclc.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -ev
+
+cd ${TRAVIS_BUILD_DIR}
```

Comment:
Don't depend on `TRAVIS_BUILD_DIR` here. Use a generic `BUILD_ROOT` environment variable. You should also check variables you intend to use are set. There's a nice concise syntax for doing this in bash

```bash
: ${BUILD_ROOT?"BUILD_ROOT is not specified}
```